src/polarization.jl

```diff
@@ -0,0 +1,48 @@
+#################
+# polarization vectors
+# 
+# In this file, we provide an extension of the Polarizations of Photons from
+# QEDbase to define polarization states of background fields
+################
+#
+# TODO:
+#   * write doc strings
+#   * implement elliptical/circular polarisation
+
+"""
+    polarization_vector(pol::AbstractPolarization, mom::QEDbase.AbstractFourMomentum)
+
+Return the polarisation vector for a given polarization and four-momentum `mom`.
+For a definite polarisation, the respective `LorentzVector` is returned, 
+where as for an indefinite polarisation, a tuple of polarisation vectors is returned.
+"""
```

Comment:
I don't know if a doc-string is the right place for such an explanation. Especially, since there is no fixed direction, where the polarization vectors need to point, but they need to fulfill $k\epsilon_i = 0$ and $\epsilon_i \epsilon_j = -\delta_{ij}$. Furthermore, we use the `base_state` function from `QEDbase`, which might change in the future. However, 224e6d8 adds a comment to indicate this choice.

Reply:
> Especially, since there is no fixed direction, where the polarization vectors need to point, ...

Well, that is exactly the point. Currently, I do not know what to expect when using this function.
Furthermore, the implementation of `base_vector` might change and its result can be different in the future.
This is not necessarily bad, yet important to know.
With the note it is at least clear now, where the behavior is defined.